pages/hpc/jobs.md

```diff
@@ -1,169 +1,257 @@
+#SBATCH --mail-type=end
+#SBATCH --partition=ezurek
+#SBATCH --qos=ezurek
+#SBATCH --cluster=faculty
+
+#Let's start some work
+echo "Hello world from faculty cluster node: "`/usr/bin/uname -n`
+#Let's finish some work
+
+```
+To submit to the **scavenger partition on the ub-hpc cluster**, the slurm script would look like:
+
+```
+#!/bin/sh
+#
+#SBATCH --time=00:01:00
+#SBATCH --nodes=1
+#SBATCH --ntasks-per-node=1
+#SBATCH --mem=10000
+#SBATCH --job-name="example-general-compute-scavenger-job"
+#SBATCH --output=example-general-compute-scavenger-job.out
+#SBATCH --mail-user=myemailaddress@institution.edu
+#SBATCH --mail-type=end
+#SBATCH --partition=scavenger
+#SBATCH --qos=scavenger
+#SBATCH --cluster=ub-hpc
+
+#Let's start some work
+echo "Hello world from ub-hpc cluster scavenger node: "`/usr/bin/uname -n`
+#Let's finish some work
+
 ```
 
 ## Useful Slurm Commands
 
 #### sbatch
-Tell me what sbatch does
+
+To submit a batch script to Slurm for processing:
+
+````
+sbatch job.slurm.script
+```` 
+For more information, [visit the Slurm docs on sbatch](https://slurm.schedmd.com/sbatch.html)
 
 #### squeue
-Tell me what squeue does
+
+To see all of your jobs in across all CCR clusters:
+````
+squeue -M all -u $LOGNAME
+````
+For more information, [visit the Slurm docs on squeue](https://slurm.schedmd.com/squeue.html)
 
 #### scancel
-Tell me what scancel does
+To cancel a job you have submitted:
+````
+scancel <job_id>
+````
+
+For more information, [visit the Slurm docs on scancel](https://slurm.schedmd.com/scancel.html)
+
+For additional Slurm commands, [visit the Slurm command manual](https://slurm.schedmd.com/quickstart.html)
+
+## Slurm <sbatch> Job Flags
+ 
+The `sbatch` command supports many optional flags. To review all the options, please visit the Slurm [sbatch page](http://slurm.schedmd.com/sbatch.html). Below are useful flags: 
+
+| Type                   | Description                                         | Flag                       |
+| :--------------------- | :-------------------------------------------------- | :------------------------- |
+| Partitions         | Specify a partition                                 | --partition=partition_name |
+| Sending email          | Receive email at beginning or end of job completion | --mail-type=type           |
+| Email address          | Email address to receive the email                  | --mail-user=user           |
+| Number of nodes        | The number of nodes needed to run the job           | --nodes=nodes              |
+| Number of tasks        | The total number of cores needed to run the job     | --ntasks=processes         |
+| Quality of service | Specify a QOS                                       | --qos=qos                  |
+| Wall time              | The max. amount of time your job will run for       | --time=wall time           |
+| Job Name               | Name your job so you can identify in queue          | --job-name=<jobname>       |
```

Comment:
The <jobname> isn't showing up in mkdocs.  Maybe the formatting is off?  It is just display --job-name=